src/Html/Abstractions/README.md

```diff
@@ -0,0 +1,21 @@
+# Html.Abstractions
+
+This project contains types for manipulating HTML content.
+
+## Description
+
+This project contains interfaces and abstractions used in MVC to support writing and modifying HTML content. For a full list of the types defined in this class, see [the namespace documentation](https://docs.microsoft.com//dotnet/api/microsoft.aspnetcore.html).
+
+## Development Setup
+
+### Build
+
+To build this specific project from source, you can follow the instructions [on building a subset of the code](https://github.com/dotnet/aspnetcore/blob/master/docs/BuildFromSource.md#building-a-subset-of-the-code).
```

Comment:
There is a mix of absolute and relative links to markdown files. I don't have a strong opinion on which one, but it is strange that there is a mix (README.md link at the bottom is relative)

Reply:
That's my bad. Fixed here and in the templates.